07_Bootstrap/praktikum/index.html

```diff
@@ -0,0 +1,161 @@
+    <link rel="stylesheet" href="./style.css">
+
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">
+    <nav class="navbar navbar-expand-lg navbar-light bg-light ">
+        <div class="container-fluid">
+          <a class="navbar-brand" href="#"><span>e</span>Startup</a>
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse justify-content-lg-end" id="navbarNavDropdown">
+            <ul class="navbar-nav">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">About</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Features</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Screenshot</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Term</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Pricing</a>
+              </li>
+              <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                  Dropdown link
+                </a>
+                <ul class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                  <li><a class="dropdown-item" href="#">Action</a></li>
+                  <li><a class="dropdown-item" href="#">Another action</a></li>
+                  <li><a class="dropdown-item" href="#">Something else here</a></li>
+                </ul>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Contact</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </div>  
+    <div class="content-heading m-3">
+        <h1 class="fw-light text-center">Simple Systeme Fordiscount</h1>
+        <h6 class="fw-lighter text-center m-4">Integer Cursus Bibendum Augue Ac Cursus</h6>
+    </div>
+    <div class="container mt-lg-5">
+    <section class="d-lg-flex" id="Features">
+        <div class="content d-lg-flex flex-column text-center">
+            <img src="./cloud.svg" alt="" class="mx-auto d-block">
+            <h6 class="fw-bolder">Introducing Whatsapp</h6>
+            <p class="fw-lighter pt-3">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Doloremque nihil magnam assumenda.</p>
+            <label>Read More</label>
+        </div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)

Reply:
baik mas arie atas sarannya